packages/storybook/stories/va-radio.stories.jsx

```diff
@@ -23,6 +23,7 @@ const Template = args => {
 };
 
 const defaultArgs = {
+  enableAnalytics: false,
```

Comment:
I think this may need to be `kebab-cased` to actually apply, like this:

https://github.com/department-of-veterans-affairs/component-library/blob/e63bb48b0063d07093d6329d9d289b352d3a31aa/packages/storybook/stories/va-loading-indicator.stories.jsx#L39

Reply:
It looks like we have the option of using camel case in JSX, but HTML must be kebab case. I think it's best if the casing is consistent across our web components. Good catch!